cpp/benchmarks/common/data_generators.cu

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "generate_input.hpp"
+#include <cudf/types.hpp>
+
+#include <rmm/cuda_stream_view.hpp>
+#include <rmm/device_buffer.hpp>
+#include <rmm/device_uvector.hpp>
+
+#include <cuda.h>
+#include <curand_kernel.h>
+
+__global__ void setup_kernel(curandState* state, int size, unsigned seed)
+{
+  int id = threadIdx.x + blockIdx.x * blockDim.x;
+  /* Each thread gets same seed, a different sequence
+     number, no offset */
+  if (id < size) curand_init(seed, id, 0, &state[id]);
+}
+
+//  int8_t,  int16_t,  int32_t,  int64_t,
+// uint8_t, uint16_t, uint32_t, uint64_t
+// float,   double
```

Comment:
Probably need a little more text here on what this comment is for.

Reply:
I removed this file. Is it still showing?

Reply:
It appears to be gone now